ci/cron/monthly.yaml

```diff
@@ -11,7 +11,7 @@ trigger: none
 
 # Run on schedule: first Monday of each month at 4AM UTC
```

Comment:
Might wanna update this comment.

Reply:
Good catch, thanks! Comments are always a lie waiting to happen.